examples/kubectl.rs

```diff
@@ -185,7 +188,14 @@ impl App {
 async fn main() -> Result<()> {
     tracing_subscriber::fmt::init();
     let app: App = clap::Parser::parse();
-    let client = Client::try_default().await?;
+
+    let options = KubeConfigOptions {
+        context: app.context.clone(),
+        cluster: None,
+        user: None,
+    };
```

Comment:
this looks sensible to me. kind of highlights that perhaps `KubeConfigOptions` should have a builder so that these 4 lines becomes would become `let options = KubeConfigOptions::default().context(app.context);`.